layouts/partials/head.html

```diff
@@ -7,19 +7,20 @@
 <link rel="{{ .Rel }}" type="{{ .MediaType.Type }}" href="{{ .Permalink | safeURL }}">
 {{ end -}}
 
-{{ if and hugo.IsProduction (ne $outputFormat "print") -}}
+{{ if and (eq (getenv "HUGO_ENV") "production") (ne $outputFormat "print") -}}
```

Comment:
@jcabak - why were the changes introduced by #413 reverted here? 
@LisaFC - maybe this is the case of #611?

/cc @nate-double-u

Reply:
I'm assuming that this (retrograde) change was accidental, but I could be wrong -- and thanks for clarifying the situation @jcabak when you have a minute. Let's carry the discussion over to #652.

Reply:
Honestly i am surprised that i changed this line, you have right @chalin it was accidental. I appreciate your vigilance 🤠

Reply:
Thanks for promptly confirming @jcabak. FYI, #653 reintroduced use of `hugo.IsProduction`.